### ISSUE TITLE
新出的自动压缩支持PPYOLO系列吗？比如PPYOLO-mbv3_small

### ISSUE THREAD

**Issue by jo-dean** [PaddlePaddle/PaddleSlim#1331]
请问新出的自动压缩支持PPYOLO系列吗？比如PPYOLO-mbv3_small模型。

**Comment by yghstill** [comment]
支持，可以根据PPYOLOE-l的配置文件改写下，就可以跑，将reader中的resize shape对齐PPYOLO-mbv3_small。

我们计划会增加更多检测模型，PPYOLO系列模型后续会逐渐加入到benchmark中，有问题欢迎讨论。

**Comment by jo-dean** [comment]
您好，这是我的配置文件。
**
Global:
  reader_config: configs/ppyolo_reader.yml
  input_list: ['image', 'im_shape', 'scale_factor']
  arch: YOLO
  Evaluation: True
  model_dir: ./ppyolo_mbv3_small_coco
  model_filename: model.pdmodel
  params_filename: model.pdiparams

Distillation:
  alpha: 1.0
  loss: soft_label

Quantization:
  use_pact: true
  activation_quantize_type: 'moving_average_abs_max'
  quantize_op_types:
  - conv2d
  - depthwise_conv2d

TrainConfig:
  train_iter: 5000
  eval_iter: 1000
  learning_rate:
    type: CosineAnnealingDecay
    learning_rate: 0.00003
    T_max: 6000
  optimizer_builder:
    optimizer:
      type: SGD
    weight_decay: 4.0e-05

**
请问Distillation: Quantization: TrainConfig:这三个选项的参数我应该如何配置呢？谢谢。

**Comment by yghstill** [comment]
@jo-dean 保持你的这样设置跑下看看精度怎么样
